test/e2e/nstemplatetier_test.go

```diff
@@ -30,11 +30,16 @@ const (
 
 func TestNSTemplateTiers(t *testing.T) {
 	// given
-	hostAwait, memberAwait, _ := WaitForDeployments(t)
+	hostAwait, memberAwait, memberAwait2 := WaitForDeployments(t)
 
 	// Create and approve "testingtiers" signups
 	testingTiersName := "testingtiers"
-	testingtiers := CreateAndApproveSignup(t, hostAwait, testingTiersName, memberAwait.ClusterName)
+	testingtiers, _ := NewSignupRequest(t, hostAwait, memberAwait, memberAwait2).
+		Username(testingTiersName).
+		ManuallyApprove().
+		RequireConditions(ConditionSet(Default(), ApprovedByAdmin())...).
+		Execute().
+		Resources()
```

Comment:
you still don't set the target cluster here now wait for the MUR to be fully provisioned - are you sure that it's fine?

Reply:
You're right, I should be setting the target cluster here and also calling EnsureMUR.  Fixed.